components/GreetingButton.js

```diff
@@ -1,6 +1,13 @@
 import { View, Text, StyleSheet } from 'react-native';
 
+
 const GreetingButton = () => {
+    
+    //make a data file with the mantras and affirmations from self-care center
+    // import the data file here
+    // create function to randomly pick a phrase from the data file and populate the greeting button
+    // try to create the data file into an API
```

Comment:
This has been accomplished. Need to pull it in.

Reply:
Woohoo! Awesome job with this, both of you!